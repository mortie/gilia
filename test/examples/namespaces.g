obj := {
	foo: "hello world"
}
print obj.foo

# Assignments
obj.foo = 100
print obj.foo

# Nested namespaces should work
obj.bar = {baz: "how's your day going?"}
print obj.bar.baz

# Empty namespace literal (should parse to namespace, not function)
print {}
