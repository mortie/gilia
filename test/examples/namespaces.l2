obj := {
	foo: "hello world"
}
print obj.foo

obj.foo = 100
print obj.foo

obj.bar = {baz: "how's your day going?"}
print obj.bar.baz

print {}
