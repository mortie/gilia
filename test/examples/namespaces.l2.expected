hello world
100
how's your day going?
(namespace)
